demo-shell/src/app/components/files/files.component.ts

```diff
@@ -74,6 +74,21 @@ export class FilesComponent implements OnInit, OnChanges, OnDestroy {
 
     processId;
 
+    @Input()
```

Comment:
All those input should have the jsdoc

Reply:
this one is demo shell and other properties don't have jsdoc there

Reply:
right 👍